src/Exception/MutatorException.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Exception;
+
+use Infection\Mutator\Util\Mutator;
+
+/**
+ * @internal
+ */
+final class MutatorException extends \Exception
+{
+    /**
+     * @param string $filePath
+     * @param Mutator $mutator
+     * @param \Throwable $previous
+     *
+     * @return MutatorException
```

Comment:
`@return self` here would be more correct, but since in this case there is absolutely no additional information brought by the phpdoc I would just remove it: it is nothing but noise in this case